LiskHQ/lisk
# Extract api codebase to http_api module - Closes #2770, #2771, #2772, #2274, #2775, #2776

### What was the problem?
There was no HTTP API module separated from Chain module
### How did I fix it?
Extracted HTTP API as a module from Chain module
### How to test it?
Run all tests, especially functional tests to check if the API is working as expected.
### Review checklist

* The PR resolves #2770
* The PR closes #2771, #2772, #2274, #2775, #2776,
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated
